src/components/note/__internal__/note.spec.js

```diff
@@ -0,0 +1,149 @@
+import React from 'react';
+import { ThemeProvider } from 'styled-components';
+import { mount } from 'enzyme';
+import Note from './note.component';
+import baseTheme from '../../../style/themes/base';
+import {
+  NoteContent, InlineControl, Title, Footer, FooterContent
+} from './note.style';
+import StatusWithTooltip from './status-with-tooltip';
+import { assertStyleMatch } from '../../../__spec_helper__/test-utils';
+import Tooltip from '../../tooltip';
+
+function render(props, renderer = mount) {
```

Comment:
You've not got any other renderers in this file so you can just get rid of this `renderer` parameter and use `mount` below

Reply:
Fair point I usually set them up like this just in case future test cases are written that use different `renderer`s, but in this component I think that is very unlikely so happy to remove

Reply:
Yeah YAGNI principle!